kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/OrderProcessor.java

```diff
@@ -0,0 +1,25 @@
+package com.kodilla.good.patterns.challenges;
+
+public class OrderProcessor {
+
+    private OrderService orderService;
+    private PaymentService paymentService;
+    private ShipmentService shipmentService;
+
+    public OrderProcessor(OrderService orderService, PaymentService paymentService, ShipmentService shipmentService) {
+        this.orderService = orderService;
+        this.paymentService = paymentService;
+        this.shipmentService = shipmentService;
+    }
+
+    public void process(final OrderRequest orderRequest) {
+        orderService.order(orderRequest.getUser(), orderRequest.getProduct());
+
+        boolean isPaid = paymentService.collect(orderRequest.getUser(), orderRequest.getProduct());
+        if(isPaid) {
```

Comment:
u ciebie zawsze zaplacil .... z metody collect zwracasz zawsze true.

Reply:
Ok, przy czym tak to było zrobione w przykładzie w submodule.

Reply:
ale mozesz miec lepiej niz w przykladzie z submodulu :) W submodule chodzi o to, by cos zaprezentowac, a nie wykonac pelna implementacje :)